test/test_linalg.py

```diff
@@ -4501,7 +4498,7 @@ def _test_addmm_addmv(self, f, t, m, v, *, alpha=None, beta=None, transpose_out=
                         torch.cfloat: 1e-4, torch.cdouble: 1e-8})
     @dtypesIfCUDA(*torch.testing.get_all_complex_dtypes(),
                   *([torch.float32, torch.float64, torch.bfloat16]
-                    if TEST_WITH_ROCM else torch.testing.get_all_fp_dtypes(include_bfloat16=AMPERE_OR_ROCM)))
+                    if TEST_WITH_ROCM else torch.testing.get_all_fp_dtypes(include_bfloat16=(CUDA11 and SM53))))
```

Comment:
can you do `include_bfloat16=(TEST_WITH_ROCM or (CUDA11 and SM53))` as in other places here?

Reply:
fixed